draft-claise-semver.md

```diff
@@ -147,7 +145,7 @@ RFC-formatted document as necessary.
 ~~~~~
 * a9d7d29 (tag:2.0.0, tag:RFCXXXX) Merge branch 'v2'
 |\
-| * df5d437 Responses WGLC and IETF LC comments
+| * df5d437 Responses to WGLC and IETF LC comments
 | | 
 | * 986ebb6 (tag:2.0.0-b2) Checkpoint for hackathon
 | | 
```

Comment:
One additional bit of example text that I think would be useful to add would be to show how the structured spec could be named versus how a corresponding internet-draft could be named, if we expect there to be a correspondence between the name of, e.g.,  a YANG model, and the name of a draft that incorporates the model.